autopprof.go

```diff
@@ -65,7 +76,12 @@ var globalAp *autoPprof
 
 // Start configures and runs the autopprof process.
 func Start(opt Option) error {
-	qryer, err := newQueryer()
+	cgroupQryer, err := queryer.NewCgroupQueryer()
+	if err != nil {
+		return err
+	}
+
+	runtimeQryer, err := queryer.NewRuntimeQueryer()
```

Comment:
typo?

Reply:
No. this is normal naming of this repository